opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -285,7 +288,7 @@ def is_recording_events(self) -> bool:
         return True
 
 
-def generate_span_id():
+def generate_span_id() -> int:
```

Comment:
These don't seem to change the results of `tox -e mypy-extra`. Why do you need them? 

I'm more than happy for internal methods like this not to have type annotations.

Reply:
Why are you happier without type annotations?

Reply:
These are admittedly pretty inoffensive, I just don't think we should require annotations unless they're useful for external users, or we use them to type check our own use of the API.

Whether you think this change is net good or net bad probably depends on whether you think it's a bug or a feature that python isn't statically typed.